Assets/Demo/SimpleDemo/SimpleControls.cs

```diff
@@ -5,52 +5,35 @@
 using UnityEngine.Experimental.Input;
 
 
-[Serializable]
-public class SimpleControls : InputActionAssetReference
+public class SimpleControls
```

Comment:
I think this needs to implement `IInputActionCollection`. Otherwise we will lose some functionality and information from the asset (e.g. access to control schemes).